.github/workflows/ci.yaml

```diff
@@ -78,7 +78,18 @@ jobs:
         with:
           go-version: v1.17
       - run: make build
-      - run: ARTIFACT_DIR=/tmp/e2e PATH="${PATH}:$(pwd)/bin/" make test test-e2e
+      - run: PATH="${PATH}:$(pwd)/bin/" make test
+
+  e2e:
+    name: e2e
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: v1.17
+      - run: make build
+      - run: ARTIFACT_DIR=/tmp/e2e PATH="${PATH}:$(pwd)/bin/" COUNT=1 E2E_PARALLELISM=2 make test-e2e
```

Comment:
Why COUNT=1 ? NACK - we've already found flakes with COUNT=5

Reply:
running tests 5 times?

Can we at least split the e2e into one that does COUNT=1 and returns early, and another with COUNT=5 ?

I hate to wait.

Reply:
e2e and e2e-quick

Reply:
How long does `-count 5` take and what is the comparison to unit tests? If it makes you wait more than 5 extra minutes, I'm more keen to try fewer counts...

Reply:
Added benchmark commit. Let's see.